kusto/ingest/internal/resources/ranked_storage_account_set_test.go

```diff
@@ -0,0 +1,228 @@
+package resources
+
+import (
+	"math"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestRankedStorageAccountSet_TestDefaultRank(t *testing.T) {
+	r := newDefaultRankedStorageAccountSet()
+
+	// Add 3 accounts
+	r.registerStorageAccount("test-account-1")
+	r.registerStorageAccount("test-account-2")
+	r.registerStorageAccount("test-account-3")
+
+	accounts := r.getRankedShuffledAccounts()
+
+	if len(accounts) != 3 {
```

Comment:
Prefer using the testify library instead, there are plenty of shorthand function and it displays diffs better:

	assert.Len(t, accounts, 3)